lib/compiler/findImports.ts

```diff
@@ -1,20 +1,11 @@
-import fs from 'fs';
-import path from 'path';
-import {readFileContent} from '../utils';
+import { ImportFile } from '@resolver-engine/imports';
 
-export function findImports(libraryPath: string) {
+export function findImports(fileCache: ImportFile[]) {
```

Comment:
I would suggest renaming `fileCache` to `sources` to match the rest of the project